types/react-with-styles/lib/ThemedStyleSheet.d.ts

```diff
@@ -1,4 +1,4 @@
-interface ThemedStyleSheet {
+declare const ThemedStyleSheet: {
```

Comment:
👍 corresponds to https://github.com/airbnb/react-with-styles/blob/61b02293f7d91cdc5fe9f5d338cc30fc21e4325b/src/ThemedStyleSheet.js#L95-L106